client/server/pages/index.js

```diff
@@ -828,21 +828,6 @@ function wpcomPages( app ) {
 				res.send( renderJsx( 'support-user' ) );
 			} );
 	} );
-
-	app.get( [ '/subscriptions', '/subscriptions/*' ], function ( req, res, next ) {
-		if ( req.context.isLoggedIn ) {
```

Comment:
I think we should remove only the following condition. I am sorry, I was a bit too vague with my explanation on our call when I said we can remove the whole condition.
```ts
if ( req.context.isLoggedIn ) {
	// We want to show the old subscriptions management portal to the logged-in users, until new one in reader is developped for them
	return res.redirect( 'https://wordpress.com/email-subscriptions?option=settings' );
}
```

Now, that we want to allow logged-in users to all the subscriptions portal paths, we should extend the following condition:
```ts
if ( reqc.context.isLoggedIn || req.cookies.subkey ) {
	// If the user is logged-in, or otherwise has a subkey cookie , they are authorized to view the page
	return next();
}
```

Lastly, we still want to show the email-subscriptions landing page to logged-out users who have no subkey. Therefore, the last redirect should remain in place.